packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -191,7 +191,10 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       SET_SESSION_INFO: { internal: true, actions: 'updateSessionInfo' },
       DISCONNECT_EVENT: { internal: true, actions: 'updateShouldDisconnect' },
       SET_DOM_AND_CAMERA_DETAILS: { actions: 'setDOMAndCameraDetails' },
-      UPDATE_DEVICE_AND_STREAM: { actions: 'updateDeviceAndStream' },
+      UPDATE_DEVICE_AND_STREAM: {
+        actions: 'updateDeviceAndStream',
+        target: 'start',
```

Comment:
Info: is this setting it so that when the device and stream are updated, we return to the `start` state?

Reply:
Correct!